core/trino-main/src/main/java/io/trino/metadata/CatalogMetadata.java

```diff
@@ -106,6 +107,14 @@ public ConnectorTransactionHandle getTransactionHandleFor(CatalogName catalogNam
         throw new IllegalArgumentException("Unknown connector id: " + catalogName);
     }
 
+    public CatalogName getConnectorId(CatalogSchemaName schema)
```

Comment:
`getCatalogName`? We should remove last mentions about `ConnectorId`.

Reply:
see overloaded method name

Reply:
I see, it should be renamed too (in separate commit).

Reply:
Please don't get me wrong. With `?` I wanted to suggest a change, not require.

Reply:
> I see, it should be renamed too (in separate commit).

i don't know whether it should, or shouldn't, but since it's preexisting name (see overload, and per your comments), i choose not to include this change in this PR

thanks for suggestion though!